cluster/gce/configure-vm.sh

```diff
@@ -492,6 +492,17 @@ EOF
   fi
 }
 
+function salt-test-node-role() {
+  cat <<EOF >/etc/salt/minion.d/grains.conf
+grains:
+  roles:
+    - kubernetes-pool
+  cbr-cidr: 10.123.45.0/30
+  cloud: gce
+  test: true
```

Comment:
As discussed offline you can just add "test: true" to the file, instead of copy&pasting the whole grains.conf (see salt-set-apiserver())